momento-sdk/src/main/java/momento/sdk/ScsDataClient.java

```diff
@@ -170,6 +171,18 @@ CompletableFuture<CacheSetResponse> set(
     }
   }
 
+  CompletableFuture<CacheIncrementResponse> increment(
```

Comment:
The other methods will never throw an exception. I think there is a disagreement between our current standards and practices doc and what I heard the latest error handling guidance is. We should be consistent either way though.

Reply:
@nand4011 would you mind filing a PR to update the S&P doc to your understanding?

Reply:
We still need error handling on these increment methods to make them match the others